src/main/resources/schemas/nhinc/common/ExchangeManagement.xsd

```diff
@@ -9,65 +9,79 @@
     <xsd:import namespace="urn:gov:hhs:fha:nhinc:exchange" schemaLocation="../../exchange/exchange.xsd"/>
     <xsd:import namespace="urn:gov:hhs:fha:nhinc:exchange:directory" schemaLocation="../../exchange/organizationDirectory.xsd"/>
 
-    <xsd:complexType name="SimpleExchangeManagementResponseMessageType">
+    <xsd:complexType name="BasicRequestMessageType" >
         <xsd:sequence>
-            <xsd:element name="status" type="xsd:boolean"/>
-            <xsd:element name="message" type="xsd:string"/>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
         </xsd:sequence>
     </xsd:complexType>
-    <xsd:element name="SimpleExchangeManagementResponseMessage" type="tns:SimpleExchangeManagementResponseMessageType"/>
+    <xsd:element name="BasicRequestMessage" type="tns:BasicRequestMessageType"/>
 
-    <xsd:complexType name="SaveExchangeRequestMessageType" >
+    <xsd:complexType name="SimpleExchangeManagementResponseMessageType">
         <xsd:sequence>
-            <xsd:element name="exchange" type="exch:exchangeType"/>
-            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="status" type="xsd:boolean" minOccurs="0"/>
+            <xsd:element name="message" type="xsd:string" minOccurs="0"/>
+            <xsd:element name="url" type="xsd:string" minOccurs="0"/>
+            <xsd:element name="hcid" type="xsd:string" minOccurs="0"/>
+            <xsd:element name="aaidList" type="xsd:string" minOccurs="0" maxOccurs="unbounded"/>
         </xsd:sequence>
     </xsd:complexType>
-    <xsd:element name="SaveExchangeRequestMessage" type="tns:SaveExchangeRequestMessageType" />
+    <xsd:element name="SimpleExchangeManagementResponseMessage" type="tns:SimpleExchangeManagementResponseMessageType"/>
 
-    <xsd:complexType name="DeleteExchangeRequestMessageType" >
-        <xsd:sequence>
-            <xsd:element name="exchangeName" type="xsd:string"/>
-            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
-        </xsd:sequence>
+    <xsd:complexType name="ExchangeRequestMessageType" >
+        <xsd:complexContent>
+            <xsd:extension base="tns:BasicRequestMessageType">
+                <xsd:sequence>
+                    <xsd:element name="exchange" type="exch:exchangeType"/>
+                </xsd:sequence>
+            </xsd:extension>
+        </xsd:complexContent>
     </xsd:complexType>
-    <xsd:element name="DeleteExchangeRequestMessage" type="tns:DeleteExchangeRequestMessageType" />
+    <xsd:element name="ExchangeRequestMessage" type="tns:ExchangeRequestMessageType" />
 
-    <xsd:complexType name="ListExchangesRequestMessageType" >
-        <xsd:sequence>
-            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
-        </xsd:sequence>
+    <xsd:complexType name="ExchangeNameRequestMessageType" >
```

Comment:
Only is needed, ExchangeNameRequestMessageType or ExchangenameOptionMessageType

Reply:
ok i will my it optional